source-map-support.js

```diff
@@ -84,7 +84,7 @@ function retrieveSourceMapURL(source) {
 
   // Get the URL of the source map
   fileData = retrieveFile(source);
-  var re = /\/\/[#@]\s*sourceMappingURL=([^'"]+)\s*$/mg;
+  var re = /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
```

Comment:
It is from the convert-souce-map project
https://www.npmjs.com/package/convert-source-map
The RegExp is on the following line:
https://github.com/thlorenz/convert-source-map/blob/master/index.js#L8